src/openrct2/drawing/Drawing.h

```diff
@@ -146,7 +146,7 @@ assert_struct_size(rct_g1_element_32bit, 0x10);
 
 enum
 {
-    G1_FLAG_BMP = (1 << 0), // Image data is encoded as raw pixels (no transparency)
+    G1_FLAG_HAS_TRANSPARENCY = (1 << 0), // Image data is encoded as raw pixels (no transparency)
```

Comment:
Update comment. Image data contains transparent pixels (0xFF) which will not be rendered.